ui/src/app/streams/stream-deploy/stream-deploy.component.ts

```diff
@@ -487,7 +488,8 @@ export class StreamDeployComponent implements OnInit, OnDestroy {
     free: FormGroup, file: FormGroup, config: StreamDeployConfig
   }) {
     const propertiesString = this.toRaw(value).map((a) => `${a.key}=${a.value}`).join('\n');
-    const filename = `${value.config.id}.${new Date().getTime()}.txt`;
+    const date = moment().format('YYYY-MM-DDTHms');
```

Comment:
I would suggest a dateformat of `YYYY-MM-HHmmss` to minimize ambiguity (So that every time value is always 2 digits).